packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2831,6 +2831,8 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     VoidCallback onScrollDown,
     VoidCallback onIncrease,
     VoidCallback onDecrease,
+    VoidCallback onMoveCursorForwardByCharacter,
```

Comment:
Should this use the word "character", since it's not characters?  Or is that just because that's the TalkBack name for it?

Reply:
I am not 100% fluent in the Unicode terminology and open to naming suggestions...

Having said that, this is one meaning the [Unicode glossary](https://www.unicode.org/glossary/#character) defines for the Word Character:
> (1) The smallest component of written language that has semantic value; refers to the abstract meaning and/or shape, rather than a specific shape (see also glyph), [...]

which seems very fitting here...

Reply:
In general I would avoid the word "character" because it's hugely vague.
However in this case I think the vagueness works in our favour, since we're planning on changing the semantics from "Unicode scalar value" (sometimes ambiguously called "rune") to "extended grapheme cluster" at some point.